app/lib/webpack/create-chain.js

```diff
@@ -322,7 +322,7 @@ module.exports = function (cfg, configName) {
           ignore: [ appPaths.resolve.app('/**/.*') ].concat(
```

Comment:
This pattern must also be fixed. Also, dotfiles are ignored by default(can be enabled with `dot: true`), so we can drop this pattern instead of fixing it because it is redundant.

Reply:
Ignore the first sentence, you have already done it 🚀.

Reply:
Good point about `dot: true` - I had forgotten. Going to update as it's a better fix.

Reply:
p.s It's `dot: false` not true.